index.html

```diff
@@ -0,0 +1,181 @@
+                    <div class="project-content">
+                        <h4 class="text-title">Multi-Post Stories Gain+Glory</h4>
+                        <ul class="technologies">
+                            <li class="Ruby">Ruby on Rails</li>
+                            <li class="css">css</li>
+                            <li class="JavaScript">JavaScript</li>
+                            <li class="html">html</li>
+                        </ul>
+                        <button class="projectbutton" type="action">See Project</button>
+                    </div>
+            </div>
+            <div class="project5">
+                <img class="projectimages" src="images/project1.jpg" alt="">
+                    <div class="project-content">
+                        <h4 class="text-title">Multi-Post Stories Gain+Glory</h4>
+                        <ul class="technologies">
+                            <li class="Ruby">Ruby on Rails</li>
+                            <li class="css">css</li>
+                            <li class="JavaScript">JavaScript</li>
+                            <li class="html">html</li>
+                        </ul>
+                        <button class="projectbutton" type="action">See Project</button>
+                    </div>
+            </div>
+            <div class="project6">
+                <img class="projectimages" src="images/project1.jpg" alt="">
+                    <div class="project-content">
+                        <h4 class="text-title">Multi-Post Stories Gain+Glory</h4>
+                        <ul class="technologies">
+                            <li class="Ruby">Ruby on Rails</li>
+                            <li class="css">css</li>
+                            <li class="JavaScript">JavaScript</li>
+                            <li class="html">html</li>
+                        </ul>
+                        <button class="projectbutton" type="action">See Project</button>
+                    </div>
+            </div>
+        </div>
+        <div class="aboutme">
+            <h1 class="titleaboutme">About me</h1>
+            <p class="aboutmeparagraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+            <button class="projectbuttonresume" type="action">Get My Resume</button>
+        </div>
+        <div class="languages">
+            <img class="iconlanguages" src="images/icon-languages.png" alt="">
+            <h2 class="titlelanguages">Languages</h2>
+            <ul class="languagestechs">
+                <li class="Ruby">Ruby on Rails</li>
+                <li class="css">css</li>
+                <li class="JavaScript">JavaScript</li>
+                <li class="html">html</li>
+            </ul>
+        </div>
+        <div class="frameworks">
```

Comment:
I suggest you remove this about me `div` from the section that holds  your cards. As per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) this should be an independent `section`, using a section tag not a `div`.
<img width="1438" alt="Screenshot 2023-01-15 at 03 52 33" src="https://user-images.githubusercontent.com/48546992/212503742-f197ebf6-b872-4140-b088-13190cabba65.png">

Separating your work to logical pieces means screen readers can easily parse your site

Reply:
done